lib/slim_lint/linter/rubocop.rb

```diff
@@ -30,18 +29,10 @@ class Linter::RuboCop < Linter
     def find_lints(ruby, source_map)
       rubocop = ::RuboCop::CLI.new
 
-      original_filename = document.file || 'ruby_script'
-      filename = "#{File.basename(original_filename)}.slim_lint.tmp"
-      directory = File.dirname(original_filename)
+      filename = document.file || 'ruby_script'
```

Comment:
Not sure if this is still necessary, but there was a separate PR that acted as a follow-up which fixed a bug introduced by this line: https://github.com/brigade/haml-lint/pull/243

Probably explains why you saw such a significant improvement in runtime, since we're not running RuboCop.

Reply:
You were perfectly right. It wasn't executing RuboCop, so I had cherry-picked the other commit. Now it appears to work

before:
```
$ time rake slim_lint
The factory_girl gem has been deprecated and has been replaced by factory_bot. Please switch to factory_bot as soon as possible.
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.2.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
app/views/test.html.slim:2 [W] RuboCop: Lint/UselessAssignment: Useless assignment to variable - `unused_variable`.
rake aborted!
slim-lint failed with exit code 65

real	0m20.146s
user	0m11.143s
sys	0m8.687s

```

after:
```
$ time rake slim_lint
The factory_girl gem has been deprecated and has been replaced by factory_bot. Please switch to factory_bot as soon as possible.
warning: parser/current is loading parser/ruby24, which recognizes
warning: 2.4.0-compliant syntax, but you are running 2.4.2.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
app/views/test.html.slim:2 [W] RuboCop: Lint/UselessAssignment: Useless assignment to variable - `unused_variable`.
rake aborted!
slim-lint failed with exit code 65

real	0m7.203s
user	0m5.737s
sys	0m1.344s
```

Unfortunately, coverage decresed because specs are not covering `'ruby_script.rb'`

Reply:
Same on haml-lint: https://coveralls.io/builds/13713394/source?filename=lib%2Fhaml_lint%2Flinter%2Frubocop.rb#L43

What is that line for?

Reply:
About performance improvement: I think that it is also due to gitx and textmate constantly seeing new files in the repo. Or my SSD is going to die. Anyway, also with gitx and textmate closed (`real	0m18.525s user	0m10.309s sys	0m8.086s`), performance improvement for stdout RuboCop checking is much better